src/server/services/wfs/qgswfsgetfeature.h

```diff
@@ -77,6 +77,8 @@ namespace QgsWfs
    */
   void writeGetFeature( QgsServerInterface *serverIface, const QgsProject *project, const QString &version, const QgsServerRequest &request, QgsServerResponse &response );
 
+  QString getSrsNameFromVersion( const QgsCoordinateReferenceSystem &crs );
```

Comment:
```suggestion
  QString getSrsNameFromVersion( const QgsCoordinateReferenceSystem &crs ) const;
```

Reply:
This can't be done. This is a `struct`, not a `class`.

Reply:
@troopa81: I was mistaken in my previous comment. What I meant is: "this can't be done. This not a `struct` or a `class`.